src/__test__/_components/AddAssetSpecContainer.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import expect from 'expect';
+import AddAssetContainer from '../../_components/AssetSpecs/AddAssetSpecContainer';
+
+const props = {
```

Comment:
Why is props defined in the global scope?

Reply:
This was an oversight. Changing it now